spec/unit/util/tagging_spec.rb

```diff
@@ -88,6 +88,13 @@
       expect(tagger).to be_tagged("goodbye")
     end
 
+    it "downcases tag arguments" do
+      tagger.tag("hello")
+      tagger.tag("goodbye")
+      expect(tagger).to be_tagged(:HEllO)
+      expect(tagger).to be_tagged("GooDByE")
+    end
+
```

Comment:
I can add that but please note that the test that precedes this one tests the opposite case, i.e. that uc tags are downcased. Also in this test, the GooDByE contains uc characters. So what would an additional uc word test add?